test/fixtures/rules/std_rule_cases/L018.yml

```diff
@@ -11,28 +11,19 @@ test_pass_with_clause_closing_oneline:
   # with statement oneline
   pass_str: with cte as (select 1) select * from cte
 
-test_fail_with_clause_closing_misaligned_indentation:
+test_pass_with_clause_closing_misaligned_indentation:
   # Fix with statement indentation
-  fail_str: |
+  pass_str: |
     with cte as (
         select 1
         ) select * from cte
```

Comment:
To me this is a regression. This should fail IMHO and be fixed.

Instead of aligning with the `WITH` should we align with wherever the CTE opening bracket line starts? (which in this case is at the start of the `WITH`, but in your templated example might not be.).

Reply:
IIUC, this PR is updating L018 so it doesn't concern itself with indentation -- it leaves that job to L003. Now L018 is **only** responsible for making sure the closing bracket of a CTE definition is on a separate line.

Reply:
Ignore. L003 will handle this. Resolving.